django-server/server-env-setup.sh

```diff
@@ -1,12 +1,9 @@
 #!/bin/bash
-# This script assumes you have python3, pip and virtualenv installed
-# (Sometimes you have to apt-get install python3-venv)
-# https://packaging.python.org/guides/installing-using-pip-and-virtual-environments/
-echo "Starting installation"
-python3 -m venv serverenv
-source serverenv/bin/activate
-pip install django
-pip install django-environ
-pip install djangorestframework
+# This script assumes you have python3, pip and pipenv installed
+# you can install pipenv using pip3 install pipenv
+echo "Starting installation of virtual environment"
+cd server
+pipenv install
```

Comment:
Is it possible to configure virtualenv name? To me it seems env will use some default.

Reply:
why do you need it? let's see https://github.com/pypa/pipenv/issues/1226. to use env later just type pipenv shell

Reply:
We should minimize changes. This one seems fine.